MoSDeF-Installation/create_github_account.md

```diff
@@ -0,0 +1,16 @@
+# Welcome to CyberCamp 2022
+
+Below is the instruction to help you create a GitHub account.
```

Comment:
May need to modify the header

Reply:
this file is getting deleted, it's just leftover from the old stuff I pulled this from!